smartrouter/image.yaml

```diff
@@ -0,0 +1,45 @@
+name: "jboss-bpmsuite-7/bpmsuite70-smartrouter"
+description: "Red Hat JBoss BPM Suite 7.0 Smart Router container image"
+version: "7.0.0"
+from: "jboss/openjdk18-rhel7:1.0"
+user: 185
+envs:
+    - name: "JBOSS_PRODUCT"
+      value: "bpmsuite-smartrouter"
+    - name: "JBOSS_BPMSUITE_SMARTROUTER_VERSION"
+      value: "7.0.0"
+    - name: "PRODUCT_VERSION"
+      value: "7.0.0"
+    - name: "KIE_ROUTER_DISTRIBUTION_ARTIFACT"
+      value: "kie-server-router-proxy-7.1.0.Beta3-redhat-1.jar"
```

Comment:
It's used by both the `install` and the `start` script and I didn't want to hardcode the name of the artifact in both scripts (it could lead to problems if when the artifact name changes)